src/lib/components/event/event-summary-row.svelte

```diff
@@ -90,17 +90,27 @@
     >
   </td>
   <td class="text-left">
-    <p class="break-word truncate text-sm md:whitespace-normal md:text-base">
+    <div class="flex flex-col gap-0">
       {#if showElapsedTimeDiff}
-        {formatDistanceAbbreviated({
-          start: initialItem.eventTime,
-          end: currentEvent.eventTime,
-        })}
-        {timeDiffChange}
+        <p class="break-word truncate text-sm md:whitespace-normal">
+          {descending ? '-' : '+'}{elapsedTime}
```

Comment:
To avoid a `-` without a time

<img width="441" alt="Screenshot 2023-07-25 at 11 26 36 AM" src="https://github.com/temporalio/ui/assets/15069288/4755e9e4-bdca-4a62-b4f3-24c689e8201b">

maybe we can do something like this?
```suggestion
          {#if elapsedTime}
            {descending ? '-' : '+'}{elapsedTime}
          {/if}
```

Reply:
👍